src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -51,12 +59,25 @@ public partial class HubConnection
         public TimeSpan ServerTimeout { get; set; } = DefaultServerTimeout;
         public TimeSpan HandshakeTimeout { get; set; } = DefaultHandshakeTimeout;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="HubConnection"/> class.
+        /// </summary>
+        /// <param name="connectionFactory">The connection factory.</param>
+        /// <param name="protocol">The protocol.</param>
+        /// <param name="serviceProvider">The service provider.</param>
```

Comment:
`An [IServiceProvider] containing the services provided to this connection.`

(I'm just going to use `[Token]` syntax to denote where I think a `<see/>` would be appropriate)